src/map.c

```diff
@@ -1,754 +1,612 @@
-/*
- * rv32emu is freely redistributable under the MIT License. See the file
- * "LICENSE" for information on usage and redistribution of this file.
- */
-
-#include <assert.h>
-#include <stdint.h>
-#include <stdlib.h>
-#include <string.h>
-
 #include "map.h"
+#include <memory.h>
```

Comment:
Include `<assert.h>` here.

Reply:
The order of header inclusion is
* System-wide headers appear first;
* Implementation-specific ones come later.